lib/roo_on_rails/default.env

```diff
@@ -9,3 +9,8 @@ NEW_RELIC_DEVELOPER_MODE=false
 NEW_RELIC_LICENSE_KEY=override-me
 RACK_SERVICE_TIMEOUT=10
 RACK_WAIT_TIMEOUT=30
+SIDEKIQ_ENABLED=true
+SIDEKIQ_THREADS=5
+SIDEKIQ_DB_POOL_SIZE=6
+SIDEKIQ_DB_REAPING_FREQUENCY=10
```

Comment:
`SIDEKIQ_THREADS` should really be 25 (or not set) as that's Sidekiq's normal default.

Ideally `SIDEKIQ_DB_POOL_SIZE` should be `SIDEKIQ_THREADS + 1` -- it's set in the `Procfile` so maybe we can do maths in there? Does something like `$(SIDEKIQ_THREADS + 1)` work in those?

`SIDEKIQ_DB_REAPING_FREQUENCY` sets `DB_REAPING_FREQUENCY` in the `Procfile` as @danielcooper says. The reason we don't set it as a global thing is that it's not recommended to be used for web processes, only for background processes.